tensorflow/core/ops/math_ops.cc

```diff
@@ -1807,50 +1807,70 @@ product: Pairwise cross product of the vectors in `a` and `b`.
 REGISTER_OP("Cumsum")
     .Input("x: T")
     .Input("axis: int32")
+    .Attr("exclusive: bool")
```

Comment:
@girving, this op was just added, is it safe to add a new attr without a default?